profiles/models.py

```diff
@@ -94,6 +94,8 @@ def __str__(self):
     country = CountryField(blank_label='(select country)', default='US')
 
     email_enabled = models.BooleanField(default=True)
+    # team to display on their profile
+    feat_team = models.ForeignKey('teams.Team', related_name='profile_feat_team', blank=True, null=True, on_delete=models.SET_NULL)
```

Comment:
It's a nitpick - but I don't usually like abbreviated variable names. We don't pay-by-the-byte these days so may as well make it `featured_team`.